ibm/mas_devops/roles/nvidia_gpu/templates/clusterpolicy.yml.j2

```diff
@@ -4,153 +4,66 @@ kind: ClusterPolicy
 metadata:
   name: gpu-cluster-policy
 spec:
-  daemonsets:
```

Comment:
@dclain I see you are updating role to worth with OCP 4.10... question: don't we need to support 4.8 as well as MAS still support it? Because if so, I presume the policies, versions and namespaces used before must still be supported right? Unless MVI claim support only for OCP 4.10?

Reply:
yeah I think we possibly need templates per OCP version I guess... cause the cluster policy will also changes for OCP 4.11. Any existing roles we can piggy back on that shows different roles/templates per OCP version?

Reply:
@dclain I'd rather we looked it up, but it seems the other roles that rely on this took the easy option:

```
# We need to know what the OCP_RELEASE is to know what channel to target the subscriptions at
ocp_release: "{{ lookup('env', 'OCP_RELEASE') | default('4.10', True) }}"
```

I suggest for now do the same, and when this is hopefully improved in the future it can be improved across the collection as a whole.